### DISCUSSION TITLE
Welcome to Exametrika Discussions!

### DISCUSSION THREAD

**Discussion by kosugitti** [kosugitti/exametrika#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phys-ken** [comment]
高校教員です。潜在ランク理論を活用した信頼性の(比較的)高い相互評価の手法について研究したいと考え、test data engineeringを頑張って読み進めています。
Rで実行できて、感動しています。CTTやIRTは定期試験の分析などでも使っています。
一方的な質問・要望が多くなってしまうかもしれませんが、可能な限りでご助言いただけると助かります。今後ともよろしくお願いします。

**Reply by kosugitti** [reply]
Welcomeです〜！いつでもどうぞ
